core/src/main/java/com/backblaze/b2/client/B2StorageClientWebifier.java

```diff
@@ -121,6 +122,9 @@ B2DownloadAuthorization getDownloadAuthorization(B2AccountAuthorization accountA
     B2FileVersion getFileInfo(B2AccountAuthorization accountAuth,
                               B2GetFileInfoRequest request) throws B2Exception;
 
+    B2FileVersion getFileInfo(B2AccountAuthorization accountAuth,
```

Comment:
so far, i've tried to overload method names only when they do the same thing and i think this is sufficiently different than the other one that it deserves a separate name.  it's info about the latest version, not a specific version, and it takes the bucket's name, too.

i propose changing this getFileInfoByName().   what do you think?

Reply:
Done.